StructureDefinition/England-Organization.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="England-Organization" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/England-Organization" />
+  <version value="1.0.0" />
+  <name value="EnglandOrganization" />
+  <title value="England Organization" />
+  <status value="draft" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
+		<name value="NHS England" />
+		<telecom>
+			<system value="email" />
+			<value value="interoperabilityteam@nhs.net" />
+			<use value="work" />
+			<rank value="1" />
+		</telecom>
+  </contact>
+  <description value="This profile defines the NHS England constraints and extensions on the UK Core FHIR resource [UK Core Organization]https://simplifier.net/hl7fhirukcorer4/ukcore-organization)." />
```

Comment:
Profile description: This profile defines the NHS England constraints and extensions on the FHIR UKCore resource

Reply:
FHIR UK Core ;)